ports/bsd/datetime-init.c

```diff
@@ -86,5 +101,6 @@ bool datetime_local(BACNET_DATE *bdate,
  */
 void datetime_init(void)
 {
+    handler_timesync_offset_set(0);
```

Comment:
should the offset be configured per project instead of per port?

Reply:
Thanks. You are right.